autorosters/autorosters_main.py

```diff
@@ -0,0 +1,270 @@
+from mwrogue.esports_client import EsportsClient
+from mwrogue.auth_credentials import AuthCredentials
+import math
+
+
+class AutoRostersRunner(object):
+    def __init__(self, site: EsportsClient, overview_page, tabs):
+        self.site = site
+        self.overview_page = self.site.cache.get_target(overview_page)
+        self.tabs = tabs
+        self.match_data = {}
+        self.alt_teamnames = {}
+        self.rosters_data = {}
+        self.PAGE_HEADER = "{{{{Tabs:{}}}}}{{{{TOCFlat}}}}"
```

Comment:
nitpick: constants like this can be declared before __init__

```
class AutoRostersRunner:
    PAGE_HEADER = "{{{{Tabs:{}}}}}{{{{TOCFlat}}}}"
    etc

   def __init__
```

etc

this is generally done for cleaner style but absolutely unimportant for the actual running of your code